.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ workflows:
 jobs:
   contract_cw721_base:
     docker:
-      - image: rust:1.58.1
+      - image: rust:1.63.0
```

Comment:
By changing this, you change the **minimal** Rust compiler version supported here. Is this really what you want? Using something like 1.60.0 ensures to not disrupt devs that don't work on latest stable.

Reply:
Yeah good point, will change